integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -307,6 +329,350 @@ public void testUpdateDataSourceInDomainUsingAuxiliaryImage() {
+  public void testErrorPathDomainWithFailCustomMountCommand() {
+
+    OffsetDateTime timestamp = now();
+
+    // get the creation time of the admin server pod before patching
+    LinkedHashMap<String, OffsetDateTime> pods = new LinkedHashMap<>();
+    pods.put(adminServerPodName, getPodCreationTime(domainNamespace, adminServerPodName));
+    // get the creation time of the managed server pods before patching
+    for (int i = 1; i <= replicaCount; i++) {
+      pods.put(managedServerPrefix + i, getPodCreationTime(domainNamespace, managedServerPrefix + i));
+    }
+
+    Domain domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource ");
+    assertNotNull(domain1.getSpec().getServerPod().getAuxiliaryImages(),
+        domain1 + "/spec/serverPod/auxiliaryImages is null");
+
+    List<AuxiliaryImage> auxiliaryImageList = domain1.getSpec().getServerPod().getAuxiliaryImages();
+    assertFalse(auxiliaryImageList.isEmpty(), "AuxiliaryImage list is empty");
+
+    // patch the first auxiliary image
+    String searchString = "\"/spec/serverPod/auxiliaryImages/0/command\"";
+    StringBuffer patchStr = new StringBuffer("[{");
+    patchStr.append("\"op\": \"add\",")
+        .append(" \"path\": " + searchString + ",")
+        .append(" \"value\":  \"exit 1\"")
+        .append(" }]");
+    logger.info("Auxiliary Image patch string: " +  patchStr);
+
+    V1Patch patch = new V1Patch((patchStr).toString());
+
+    boolean aiPatched = assertDoesNotThrow(() ->
+            patchDomainCustomResource(domainUid, domainNamespace, patch, "application/json-patch+json"),
+        "patchDomainCustomResource(Auxiliary Image)  failed ");
+    assertTrue(aiPatched, "patchDomainCustomResource(auxiliary image) failed");
+
+    // check the introspector pod log contains the expected error msg
+    String expectedErrorMsg = "Auxiliary Image: Command 'exit 1' execution failed in container";
+    String introspectorPodName = assertDoesNotThrow(() -> getIntrospectorPodName(domainUid, domainNamespace));
+    checkPodLogContainsString(domainNamespace, introspectorPodName, expectedErrorMsg);
+
+    // check the domain event contains the expected error msg
+    checkDomainEventContainsExpectedMsg(opNamespace, domainNamespace, domainUid, DOMAIN_PROCESSING_FAILED,
+        "Warning", timestamp, expectedErrorMsg);
+
+    // check the operator pod log contains the expected error msg
+    String operatorPodName =
+        assertDoesNotThrow(() -> getOperatorPodName(OPERATOR_RELEASE_NAME, opNamespace));
+    checkPodLogContainsString(opNamespace, operatorPodName, expectedErrorMsg);
+
+    // verify the domain is not rolled
+    // TODO: enable this check once https://jira.oraclecorp.com/jira/browse/OWLS-90971 is fixed
+    /*
```

Comment:
Use the utility oracle.weblogic.kubernetes.utils.CommonMiiTestUtils.verifyPodsNotRolled;

Reply:
yes, in the oracle.weblogic.kubernetes.utils.CommonMiiTestUtils.verifyPodsNotRolled, it only sleeps for 2 minutes. It is not enough in this case.